tests/unit/broccoli/default-packager/styles-test.js

```diff
@@ -151,7 +155,7 @@ describe('Default Packager: Styles', function() {
     ).to.equal('body { height: 100%; }');
     expect(
       outputFiles.assets['the-best-app-ever.css'].trim()
-    ).to.equal('html { height: 100%; }');
+    ).to.equal(MODULES['the-best-app-ever'].styles['app.css']);
```

Comment:
could you revert this change and match against the actual content? makes it easier to understand w/o having to scroll up